keen/explorer
# Fixes #107

## What does this PR do? How does it affect users?

`dataType` determination was incorrect for multi-group by queries. This PR fixes that and ads test coverage.

## How should this be tested?

1) Try running multi-group by queries on master. Notice they break and do not visualize.
2) Try same on this branch. Watch the visualization appear as expected.

Step through the code line by line. Things to keep in mind as you review:
 - Are there any edge cases not covered by this code?
 - Does this code follow conventions (naming, formatting, modularization, etc) where applicable?

Fetch the branch and/or deploy to staging to test the following:


## Related tickets?
#107
